test/unit/input/anyref_target_feature.wat

```diff
@@ -0,0 +1,3 @@
+(module
+ (global $g (mut anyref) (ref.null any))
+)
```

Comment:
I don't think this file needs to be checked in.

Reply:
Removed this file in the rebased PR